src/api/controllers/credentials.ts

```diff
@@ -61,8 +61,59 @@ export class Credentials {
 		})
 	}
 
-	async issue_credentials(request: Request, user: GenericAuthUser, provider: string, subjectId?: string): Promise<Response> {
-		provider = provider.toLowerCase()
+	async issue_person_credential(user: GenericAuthUser, provider: string, subjectId?: string) {
```

Comment:
@DaevMithran add the return type on the method signature

Reply:
Sure got it, I'll change that, or is it better to remove the URL itself instead of hardcoding these?